docker-compose.dev.yml

```diff
@@ -20,7 +20,7 @@ services:
             context: .
             dockerfile: Dockerfile
             target: base-stage
-        command: 'yarn storybook'
+        command: 'yarn start-storybook --port 3000 -s public'
```

Comment:
Is this a change that needs to stay in the `docker-compose.dev.yml`  or were you just trying something and forgot to change?

Reply:
Yeah, you told me to change the command to that because otherwise, I wasn't able to open storybook on my browser. When I would go to http://localhost:3001/ I would get a "This site can't be reached" page. But if that issue is no longer in dev then I can change the command back.

Reply:
Oh, I see now why.  I had forgotten.  Yeah, just keep that in then.